config/index.js

```diff
@@ -379,23 +373,57 @@ var conf = convict({
       format: 'port',
       doc: 'Port for Redis server'
     },
-    sessionsKeyPrefix: {
-      default: 'fxa-auth-session',
-      env: 'SESSIONS_REDIS_KEY_PREFIX',
-      format: String,
-      doc: 'Key prefix for session tokens in Redis'
-    },
-    maxConnections: {
-      default: 200,
-      env: 'REDIS_POOL_MAX_CONNECTIONS',
-      format: 'int',
-      doc: 'Maximum connection count for Redis'
+    sessionTokens: {
+      enabled: {
+        default: true,
+        doc: 'Enable Redis for session tokens',
+        format: Boolean,
+        env: 'USE_REDIS'
+      },
+      prefix: {
+        default: 'fxa-auth-session',
+        env: 'SESSIONS_REDIS_KEY_PREFIX',
+        format: String,
+        doc: 'Key prefix for session tokens in Redis'
+      },
+      maxConnections: {
+        default: 200,
+        env: 'REDIS_POOL_MAX_CONNECTIONS',
+        format: 'int',
+        doc: 'Maximum connection count for the session token Redis pool'
+      },
+      minConnections: {
+        default: 2,
+        env: 'REDIS_POOL_MIN_CONNECTIONS',
+        format: 'int',
+        doc: 'Minimum connection count for the session token Redis pool'
+      }
```

Comment:
If this PR is going to stick around, we need the ability to independently disable Redis for email config and session token updates. We also need to specify different key prefixes for each, which means having a different pool for each, which means coming up with appropriate settings for `maxConnections` and `minConnections` for each.

I could have done all that and left the old config untouched, but I was concerned that would make it more confusing to read/use. It would leave us with a `config.redis.enabled` and a `config.redis.email.enabled`, for instance. Moving the former to `config.redis.sessionTokens.enabled` more clearly marks them out as peers, rather than some kind of parent-child relationship or whatever.

Reply:
This rationale makes sense to me.